webapp/packages/core-utils/src/getUniqueConnectionName.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * cloudbeaver - Cloud Database Manager
+ * Copyright (C) 2020 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+/**
+ * Help to create unique name for connection
+ * @param  {string} baseName
+ * @param  {string[]} connectionNames
+ * @returns string
+ */
+export function getUniqueConnectionName(baseName: string, connectionNames: string[]): string {
+  let index = 1;
+  let name = baseName;
+
+  const names = new Set();
+  for (const connectionName of connectionNames) {
+    names.add(connectionName);
+  }
```

Comment:
extra step, remove it

Reply:
also, you can do `const names = new Set(connectionNames);` instead, but I don't see the real reason for that